### PR TITLE
Allow saving lossless WEBPs

### PR DESCRIPTION
**Pull request by ogmaresca**
This flag only applies to WEBPs. It is not the same as 100% quality.
A sample 1536x1536 image (2x upscaled) was:
* 382.3 KB for a 95% webp
* 598.3 KB for a 100% jpeg
* 563.9 KB for a 100% webp
* 2.9 MB for a png
* 2.1 MB for a lossless webp
The 100% quality was achieved by updating the UI to change the max from 95% to 100%.

### CODE REVIEW

File: sdkit/models/model_loader/stable_diffusion/convert_from_ckpt.py [hunk 1]
```diff
@@ -327,19 +327,19 @@ def create_depth_diffusers_config(original_config, image_size: int):
     """
     depth_params = original_config.model.params.depth_stage_config.params
 
-    block_out_channels = [vae_params.ch * mult for mult in vae_params.ch_mult]
+    block_out_channels = [depth_params.ch * mult for mult in depth_params.ch_mult]
```
Comment (ogmaresca) [inline sdkit/models/model_loader/stable_diffusion/convert_from_ckpt.py:330]: Made this change to make pylint happy

File: sdkit/models/model_loader/vae.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def load_model(context: Context, **kwargs):
             vae_config = create_vae_diffusers_config(original_config, image_size=image_size)
             vae_dict = convert_ldm_vae_checkpoint(vae, vae_config)
 
-            log.info(f"Loading diffusers vae")
+            log.info("Loading diffusers vae")
```
Comment (ogmaresca) [inline sdkit/models/model_loader/vae.py:44]: Another pylint fix

**Comment by ogmaresca** [comment]
I tried fixing pylint errors, but then more errors from convert_from_ckpt.py popped up

**Comment by cmdr2** [comment]
Thanks!

**Pull request merged**